tools/bin/capture-docker-stack

```diff
@@ -11,6 +11,11 @@ STACK_VERSION=$(echo "${STACK}" | cut -d '-' -f 2-)
 
 DOCKER_IMAGE=heroku/$STACK_NAME:$STACK_VERSION
 DOCKER_IMAGE_VERSION=$(docker inspect "${DOCKER_IMAGE}" | jq .[].Id | cut -d ':' -f 2 | cut -b 1-12)
+# Using `du` rather than the `Size` attribute from Docker inspect, since the latter appears to:
+#  - Under-report usage slightly when using the overlay2 storage driver
```

Comment:
Got a source/link for this? Or is it maybe a GB vs GiB thing? A `docker inspect` shows me `663771612`, and `du` shows `663` with `--block-size M`, that seems awfully suspicious to me. `du -sx -h` gives `663M`, too, while `du -sx --si` gives `696M`.

Reply:
I don't have the console output any more.

The containerd snapshotter issue alone is enough to warrant not using `docker inspect` (we have to use it locally if we want to build multi-arch, so it's going to become a necessity).